### ISSUE TITLE
BUG - Sev 3 - Android: After copy/paste was added, "Accessibility Scanner" became unusable due to picking up parts of containers not entire container

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va-mobile-app#1013]
## What happened?
Soon after copy/paste functionality went in, I noticed that Accessibility Scanner was no longer functioning correctly in that it wasn't picking up the entire touch point, just parts of the touch point.  I figured this was parts of the container and not the entire container.  This causes it to flag just about everything as not being large enough touch points.  All of the flagged touch points were previously passing in the tool.

## Specs:
- Device: Android with Accessibility Scanner

## Steps to Reproduce
- Use Android to display the health page
- Activate Accessibility Scanner
- Take a screen shot and notice all of the flagged touch points (actually items inside the touch points.)

## Desired behavior
Accessibility Scanner behaves correctly like it did before copy/paste functionality went in to the app.

## Acceptance Criteria
Accessibility Scanner behaves correctly like it did before copy/paste functionality went in to the app.

## Bug Severity
- 3 - Medium

## Linked to Story
department-of-veterans-affairs/va.gov-team#24558

## Screen shot(s) and additional information

![screenshot_V_A_2023-06-29-09_03_01.png](https://images.zenhubusercontent.com/63d026ec6c5e4a4120a46ffe/41792c79-45e1-4bf1-bc8e-8ee369d1d108)

**Comment by Sparowhawk** [comment]
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbkdtIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--f92f3c2600bb306699a7775bc62d81c0aaed5b47/Screenshot%202023-09-15%20at%2010.48.44%20AM.png)

Issue still hasn't been fixed. Will need to either be fine with it not working and close this ticket or we need ux or someone to make a decision on rules to apply to textviews on which ones should be copyable or not and if it would negatively impact veterans

**Comment by bischoffa** [comment]
I am going to moved this back to new issues. This ticket was reviewed as expected but needs additional direcetions. Seems like comments back from 10/2021 are still fitting. 

I have reached out to Brea to get her thoughts as it relates to accessibility needs. 

At this time no next steps to get this resolved, for the time  being accepting this bug.

**Comment by bischoffa** [comment]
@brea11y what is your thoughts on this tickets and Dylan's findings?

**Comment by brea11y** [comment]
Hey @bischoffa and @Sparowhawk – I've never personally used Accessibility Scanner myself, but if this is something that is useful for QA, ENG, and others, I'd say finding a way to make it work properly is important. I reviewed the [copy & paste documentation list](https://docs.google.com/document/d/1qPiJdtN9zjD_gJweT1YYJh7cI1TP2kL-tHVMZ1O4VAA/edit?usp=sharing) that is linked in one of the previous comments and agree with what is in there. I don't see a negative impact to Veterans since we would be allowing them to copy and paste into the different form fields where that functionality would be helpful and likely needed (with form fields being the most important).

**Comment by TKDickson** [comment]
We're mostly done with the copy-paste-change related issues with accessibility scanner; it still does have issues with how our screens are layered on top of each other (ex: the box in the center of this screenshot is for a component on the screen behind the large panel, not the large panel itself): <img width="364" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMWRrQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--397e84e12f9adf64a8ae1c666b21d138b2b908d3/image.png" alt="image.png" />

I'm going to talk about the scanner app generally at the QA sync on 8/19. (Usefulness, pros/cons of using it for our app, whether we want to create a spike around it for an upcoming sprint, etc)

**Comment by TKDickson** [comment]
Confirmed with QART that we do not want/require this tool; closing as not fixing.
